sdk-workflows/src/test/java/io/dapr/workflows/client/DaprWorkflowClientTest.java

```diff
@@ -30,8 +26,15 @@ public void scheduleNewWorkflow() {
 
   }
 
-  @Test(expected = IllegalArgumentException.class)
-  public void scheduleNewWorkflowEmptyContext() throws IllegalArgumentException {
-    client = new DaprWorkflowClient(null);
+  @Test
+  public void terminateWorkflow() {
+    client.terminateWorkflow("TestWorkflow", null);
+    verify(client, times(1)).terminateWorkflow("TestWorkflow", null);
```

Comment:
The value in this test is verifying that the inner client has been called with those same arguments. In order to do that we will need to change the constructor itself.

Testability is why they do all that goofy stuff with constructors:
[private ActorRuntime constructor](https://github.com/mindlessroman/java-sdk/blob/master/sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntime.java#L105-L114) && corresponding [ActorRuntimeTest](https://github.com/mindlessroman/java-sdk/blob/master/sdk-actors/src/test/java/io/dapr/actors/runtime/ActorRuntimeTest.java#L103-L117) setup